### DISCUSSION TITLE
Possibly to install only a control plane or etcd role on something lightweight like a Pi?

### DISCUSSION THREAD

**Discussion by bodleytunes** [k3s-io/k3s#3163] [Q&A]
Hi,
Just was wondering if I wanted to use a Pi4 but only for a 3rd node to allow for cluster quorum, I was thinking the approach that would stress the pi the least would be if you could pick to only participate in etcd or control-plane?

Is it possible to do this so it only would act as a 3rd node for quorum while the other two more heavyweight hosts take all the workloads.

I have dabbled with noexecute on the pi which works to an extent but I can still see some core containers being created on there.  I kind of wondered if it would be possible to have no containers at all on there.

Many thanks,
Jon.

**Comment by brandond** [comment]
Etcd is in general more IO intensive than you would want to run on a Pi with a SD card. Kubernetes itself doesn't have any quorum requirements, so adding a 3rd node just for the control-plane components doesn't get you anything.
